<div class="chatlog__message-group">
  <div id="chatlog__message-container-536865141052932096" class="chatlog__message-container" data-message-id="536865141052932096">
    <div class="chatlog__message">
      <div class="chatlog__message-aside"><img class="chatlog__avatar" src="https://cdn.discordapp.com/avatars/435789556172849152/d923305b7e58e7c4af8ae60073d8a2e9.png?size=512" alt="Avatar" loading="lazy"></div>
      <div class="chatlog__message-primary">
        <div class="chatlog__header"><span class="chatlog__author" style="color:rgb(52,152,219)" title="swift40#0279" data-user-id="435789556172849152">swift40</span> <span class="chatlog__author-tag">BOT</span> <a href="/channels/375206337937801216?category=main&amp;channel=swift-zatsu&amp;message_id=536865141052932096"><span class="chatlog__timestamp"></span></a><a href="#chatlog__message-container-536865141052932096">01/21/2019 11:10 AM</a></div>
        <div class="chatlog__content chatlog__markdown"><span class="chatlog__markdown-preserve">exit status: 1 with stdout:<code class="chatlog__markdown-pre chatlog__markdown-pre--multiline nohighlight">Test Suite 'All tests' started at 2019-01-21 20:10:23.991 Test Suite 'bin.xctest' started at 2019-01-21 20:10:23.993 Test Suite 'Test' started at 2019-01-21 20:10:23.993 Test Case 'Test.testA' started at 2019-01-21 20:10:23.993 &lt;EXPR&gt;:0: error: Test.testA : threw error "my error" Test Case 'Test.testA' failed (0.004 seconds) Test Suite 'Test' failed at 2019-01-21 20:10:23.997 Executed 1 test, with 1 failure (1 unexpected) in 0.004 (0.004) seconds Test Suite 'bin.xctest' failed at 2019-01-21 20:10:23.997 Executed 1 test, with 1 failure (1 unexpected) in 0.004 (0.004) seconds Test Suite 'All tests' failed at 2019-01-21 20:10:23.997 Executed 1 test, with 1 failure (1 unexpected) in 0.004 (0.004) seconds</code></span></div>
      </div>
    </div>
  </div>
</div>